### ISSUE TITLE
Chat Sample account creation doesn't work

### ISSUE THREAD

**Issue by mikaelm12** [aspnet/SignalR#38]
The chat sample has issues with the versions of the EF packages preventing you from creating new users.
@BrennanConroy 

```
fail: Microsoft.EntityFrameworkCore.Query.Internal.SqlServerQueryCompilationContextFactory[1]
      An exception occurred in the database while iterating the results of a query.
      System.Data.SqlClient.SqlException: Cannot open database "aspnet-ChatSample-f11cf018-e0a8-49fa-b749-4c0eb5c9150b" requested by the login. The login failed.
      Login failed for user 'REDMOND\mimengis'.
         at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, Object providerInfo, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling)
         at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
         at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
         at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
         at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
         at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
         at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
      --- End of stack trace from previous location where exception was thrown ---
         at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
         at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```

**Comment by moozzyk** [comment]
Is it really package versions? The exception is just saying that the connection to the database can't be opened...

**Comment by BrennanConroy** [comment]
He didnt mention that running 'dotnet ef' fails

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by moozzyk** [comment]
So, migrations were not applied as a result there is no database hence the error at runtime if I understand correctly.

**Comment by mikaelm12** [comment]
So turns out the `Microsoft.EntityFrameworkCore.Tools` [here](https://github.com/aspnet/SignalR/blob/dev/samples/ChatSample/project.json#L52) in the tools section needs to be ` Microsoft.EntityFrameworkCore.Tool.Dotnet` and we need another reference to `Microsoft.EntityFrameworkCore.Tools`  in the dependencies section.

**Comment by BrennanConroy** [comment]
Fixed in 1a29ddb9dedfb59fb0ec0154debe23fa42e61c4d
